Src/Fido2.Models/Converters/Base64UrlConverter.cs

```diff
@@ -8,20 +12,71 @@ namespace Fido2NetLib;
 /// </summary>
 public sealed class Base64UrlConverter : JsonConverter<byte[]>
 {
+    public static bool EnableRelaxedDecoding { get; set; }
+
     public override byte[] Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {
-        if (!reader.HasValueSequence)
+        byte[]? rentedBuffer = null;
+
+        scoped ReadOnlySpan<byte> source;
+
+        if (!reader.HasValueSequence && !reader.ValueIsEscaped)
         {
-            return Base64Url.DecodeUtf8(reader.ValueSpan);
+            source = reader.ValueSpan;
         }
         else
         {
-            return Base64Url.Decode(reader.GetString());
+            int valueLength = reader.HasValueSequence ? checked((int)reader.ValueSequence.Length) : reader.ValueSpan.Length;
+
+            Span<byte> buffer = valueLength <= 32 ? stackalloc byte[32] : (rentedBuffer = ArrayPool<byte>.Shared.Rent(valueLength));
+            int bytesRead = reader.CopyString(buffer);
+            source = buffer[..bytesRead];
+        }
+
+        try
+        {
+            return Base64Url.DecodeFromUtf8(source);
+        }
+        catch
+        {
+            if (Base64.IsValid(source))
+            {
+                if (EnableRelaxedDecoding)
+                {
+                    return Base64Url.DecodeFromUtf8(source);                    
```

Comment:
This should perhaps be `Base64.Decode...`?

Reply:
DUH. Let me also add a test case verifying this behavior.